API/Services/SeriesService.cs

```diff
@@ -485,8 +485,7 @@ public async Task<SeriesDetailDto> GetSeriesDetail(int seriesId, int userId)
             if (v.Number == 0) return c;
             c.VolumeTitle = v.Name;
             return c;
-        })).ToList();
-
+        }).OrderBy(c => float.Parse(c.Number), new ChapterSortComparer()));
```

Comment:
The volumes are already sorted, so this sorts the issues with a volume. SelectMany will present that order when merging the list of issues from each volume.